docs/README.md

```diff
@@ -14,7 +14,7 @@ Project Docs
 - [Strong Name Signing](https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/strong-name-signing.md)
 - [Public Signing](https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/public-signing.md)
```

Comment:
While we are in here.... can we update these to point to `dotnet/runtime`?

Reply:
Yep. They are now updated.